pkg/apis/forklift/v1beta1/provider.go

```diff
@@ -56,10 +51,8 @@ func (t ProviderType) String() string {
 	return string(t)
 }
 
-// Secret fields.
 const (
-	Token    = "token"
-	Insecure = "insecureSkipVerify"
+	Insecure = "insecure"
```

Comment:
why changing this? we may have existing secret with insecureSkipVerify

Reply:
don't know, I tried to restore it and this is what happened